libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/MapboxTripStarter.kt

```diff
@@ -179,6 +179,7 @@ class MapboxTripStarter internal constructor(
             replayRouteSession = null
             replayHistorySession.onDetached(mapboxNavigation)
             mapboxNavigation.startTripSession()
+            mapboxNavigation.resetTripSession { }
```

Comment:
It's not blocking. Is it expected?

Reply:
blocking the upstream flow could provide better support for rapid type changes. this change was a quick fix for a crash